web3/providers/eth_tester/main.py

```diff
@@ -30,12 +30,22 @@ class EthereumTesterProvider(BaseProvider):
     api_endpoints = None
 
     def __init__(self, ethereum_tester=None, api_endpoints=None):
+        # do not import eth_tester until runtime, it is not a default dependency
+        from eth_tester import EthereumTester, PyEVMBackend
         if ethereum_tester is None:
-            # do not import eth_tester until runtime, it is not a default dependency
-            from eth_tester import EthereumTester
             self.ethereum_tester = EthereumTester()
         else:
-            self.ethereum_tester = ethereum_tester
+            if isinstance(ethereum_tester, EthereumTester):
+                self.ethereum_tester = ethereum_tester
+            elif isinstance(ethereum_tester, PyEVMBackend):
+                self.ethereum_tester = EthereumTester(ethereum_tester)
```

Comment:
I find it a little odd to be able to pass the tester or the backend, but no objection :+1: